packages/dev/core/src/node.ts

```diff
@@ -125,6 +125,14 @@ export class Node implements IBehaviorAware<Node> {
      */
     public inspectableCustomProperties: IInspectable[];
 
+    /**
+     * Gets or sets the accessibility tag to describe the mesh for accessibility purpose.
+     */
+    public accessibilityTag?: IAccessibilityTag = {

```

Comment:
instead of initializing it with a false value, maybe this should be Nullable and set to null? this way you don't need the isSalient flag at all

Reply:
Good idea! Let me change it.